src/components/profileEdit/ProfileInfo.tsx

```
@@ -116,321 +126,303 @@ export default function ProfileInfo() {
-    useEffect(() => {
-        setNewName(profile.name)
-        setProfileImage(profile.imageUrl)
-    }, [profile])
-
-    useEffect(() => {
-        setNewIntroduction(intro.intro)
-        setNewUrls(intro.urls)
-    }, [intro])
+        setProfile({
+            ...profile,
+            nickname: prevProfile.nickname,
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: loadData

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)